src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
You are using BEM methodology for your class naming which is good. However, the modifiers `calendar--start-day-sun` and `calendar--month-length-31` seem to be static. You should consider making them dynamic to reflect the current month and start day.